packages/blaze/template.js

```diff
@@ -56,6 +62,52 @@ Blaze.isTemplate = function (t) {
   return (t instanceof Blaze.Template);
 };
 
+/**
+ * @name  onCreated
+ * @instance
+ * @memberOf Template
+ * @summary Add a callback when an instance of a template is created.
```

Comment:
This summary is incorrect - it doesn't add the callback when the template is created. It add a callback that is called when an instance of a template is created. So maybe: "Register a function to be called when an instance of this template is created."